protocol/protobuf/profile_showcase.proto

```diff
@@ -17,20 +17,33 @@ message ProfileShowcaseAccount {
 }
 
 message ProfileShowcaseCollectible {
-  string uid = 1;
+  string uid = 1 [deprecated = true];
```

Comment:
Just curious, why is it deprecated ?

Reply:
I originally used a synthetic identifier which is `chainID+contractAddress+TokenID`, but since the wallet has an api to get specific collections on contractAddress, tokenID, chainID and wallet account address, it is better to keep these fields separate. But yes, there is no sense in depreciation here, i will just remove it